estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "This is a estate property"
+    _order = "sequence"
```

Comment:
That `_order` will define how the `search` will return you the records.
You don't have a `sequence` field on your `estate.property` model, leave it empty to keep the default (that order by ID) for now.